bitnami/redis-cluster/7.4/debian-12/prebuildfs/opt/bitnami/scripts/liblog.sh

```diff
@@ -69,7 +92,32 @@ warn() {
 #   None
 #########################
 error() {
-    log "${RED}ERROR${RESET} ==> ${*}"
+    local msg_color="$RED"
+    local reset_color="$RESET"
+    local color_check="${BITNAMI_COLOR:-true}"
+    if ! [[ "$color_check" =~ ^(yes|true)$ ]]; then
+        msg_color=""
+        reset_color=""
+    fi
+    log "${msg_color}ERROR${reset_color} ==> ${*}"
+}
+########################
+# Log an 'errorX' message
+# Arguments:
+#   Message to log
+# Returns:
+#   None
+#########################
+errorX() {
+    local msg_color="$RED"
+    local reset_color="$RESET"
+    local color_check="${BITNAMI_COLOR:-true}"
+    if ! [[ "$color_check" =~ ^(yes|true)$ ]]; then
+        msg_color=""
+        reset_color=""
+    fi
+    log "${msg_color}ERROR${reset_color} ==> ${*}"
+    exit 1
```

Comment:
Notice this is a logging library, and that running `exit 1` here doesn't make much sense. Please remove this method.

Reply:
Do not disregard this comment, please.